pages/team.js

```diff
@@ -362,6 +433,12 @@ and participating in academic competitions with his friends.`}
                 text="Amogh is all about having fun. Whether it’s running an art showcase on the Slack or massive Kahoots at hackathons, he loves to run awesome events. Amogh joined HQ to run spectacular community events, to make Hack Club the best place to be a teenager on the internet, and as Hack Club’s second best rapper."
                 pronouns="he/him"
               />
+              <Bio
+                name="Clippy"
+                teamRole="Community"
+                text="A friendly bot that assists new members with joining Hack Club."
+                img="/team/clippy.jpg"
+              />
```

Comment:
Could we put Clippy in the Bots section but mention that they are retired? It feels weird to have it in the same section as human staff.

Reply:
yes that makes sense, do you have any ideas for the role? maybe Greeter (retired)?

Reply:
sounds good

Reply:
done, thank you!